setup.cfg

```diff
@@ -42,7 +42,7 @@ ignore = E501,E741,W503,E203,E261
 
 [flake8]
 max-line-length = 88
-ignore = E501,E741,W503,E203,E261,F401,F841
+ignore = E501,E741,W503,E203,E261,F401,F841,E999
```

Comment:
We would we want to ignore invalid syntax? https://github.com/PyCQA/pycodestyle/issues/584
Please try to not ignore useful errors here.

Reply:
I am also not a fan of other error messages that you are suppressing here such as `F841`, which hides assigned but unused variables - this often reveals typos.

Reply:
E999 is buggy reporting the trailing comma https://github.com/hase-project/hase/blob/master/hase/symbex/start_state.py#L60 (with pypy3.5-6.0.0)

Removing the trailing comma seems to fix it. 

F401 problem is that when type hinting in comment, the type is considered as unused module.

I will remove F841.

Reply:
For pypy3.5, variable type declaration is invalid syntax